modules/API/external/external_api.js

```diff
@@ -347,6 +367,25 @@ JitsiMeetExternalAPI.prototype.removeEventListeners = function(events) {
 };
 
 /**
+ * Returns the number of participants in the conference.
+ * NOTE: the local participant is included.
+ * @returns {int} the number of participants in the conference.
+ */
+JitsiMeetExternalAPI.prototype.getNumberOfParticipant = function() {
```

Comment:
`getNumberOfParticipant` should probably have an `s` at the end to become `getNumberOfParticipants`.